src/env.js

```diff
@@ -25,21 +25,27 @@
 import fs from 'fs';
 import path from 'path';
 import url from 'url';
+import { Buffer } from 'buffer';
 
 import { ONNX } from './backends/onnx.js';
 const { env: onnx_env } = ONNX;
 
-
 // Check if various APIs are available (depends on environment)
+const IS_REACT_NATIVE = typeof navigator !== 'undefined' && navigator.product === 'ReactNative';
 const WEB_CACHE_AVAILABLE = typeof self !== 'undefined' && 'caches' in self;
-const FS_AVAILABLE = !isEmpty(fs); // check if file system is available
+const FS_AVAILABLE = !isEmpty(fs) || IS_REACT_NATIVE; // check if file system is available
 const PATH_AVAILABLE = !isEmpty(path); // check if path is available
 
+const GCANVAS_AVAILABLE = IS_REACT_NATIVE && typeof document !== 'undefined' && typeof Image !== 'undefined';
```

Comment:
Rather do not perform check here. `env` should only be used for cases where the user can modify something

Better to move this to image.js since it's only ever used there.

Reply:
https://github.com/xenova/transformers.js/blob/185c58fd19681593fa4dcc32795145786ce6a378/src/env.js#L110

I used here.

User can disable it even he has gcanvas (It not really stable)